app/models/concerns/rule_remediation.rb

```diff
@@ -4,11 +4,15 @@
 module RuleRemediation
   extend ActiveSupport::Concern
 
+  # It is recommended to prefecth profiles with their benchmarks:
+  #
+  #  `.includes(profiles: :benchmark)`
+  #
```

Comment:
I'm not personally a huge fan of comments like this; Rails will warn of the N+1 query if you don't use `includes`, but if you want to keep it:

```suggestion
  # It is recommended to prefetch profiles with their benchmarks:
  #
  #  `.includes(profiles: :benchmark)`
  #
```

Reply:
Noted, and fixed.